### PR TITLE
Feature/110 add empty&loading view

### LINKED ISSUE #110: Empty/Loading view for Tips, Council & Treasury
maybe refactor `useTips.ts`  to use `useAsyncFetch` to have a loading state and have simpler code.

### PR DESCRIPTION
**Pull request by ahoseinian**
fixes #110

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   plugins: ['@typescript-eslint'],
   rules: {
+    'no-undef': 0,
```
Comment (geastwood) [inline .eslintrc.js]: Why to turn off this?

File: src/presentational/EmptyView.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import {StyleSheet, View} from 'react-native';
+import React from 'react';
+
+export function EmptyView() {
+  const themeVars = useTheme();
+
+  return (
+    <View style={styles.container}>
+      <Icon
+        style={{width: 32, height: 32, marginRight: 10}}
```
Comment (geastwood) [inline src/presentational/EmptyView.tsx]: Let's put them in `styles`

File: .eslintrc.js (hunk 1 above)
Comment (ahoseinian) [reply .eslintrc.js]: I couldn't get the error to go away, but after adding `plugin:@typescript-eslint/recommended` it's fixed now

File: src/presentational/EmptyView.tsx (hunk 2 above)
Comment (ahoseinian) [reply src/presentational/EmptyView.tsx]: done

**Review by geastwood — APPROVED** [review]
LGTM, ty

**Pull request merged**